From 9a0256dba4ba612ae2b4e2a925103c62942908ba Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Fri, 20 Jan 2017 10:07:59 +0800
Subject: [PATCH] clk: rockchip: rk3399: Mark some grf clock as critical

pclk_perihp_grf and pclk_vio_grf is for some grf regs read and write,
mark it as critical and it never turns off.

Change-Id: If9465334b9168b4376a7ac95d5f08e389048409f
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk3399.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/clk/rockchip/clk-rk3399.c b/drivers/clk/rockchip/clk-rk3399.c
index f1fc14863904..d1ce2c56f56f 100644
--- a/drivers/clk/rockchip/clk-rk3399.c
+++ b/drivers/clk/rockchip/clk-rk3399.c
@@ -1654,6 +1654,8 @@ static const char *const rk3399_cru_critical_clocks[] __initconst = {
 	/* ddrc */
 	"sclk_ddrc"
 	"pclk_vio",
+	"pclk_vio_grf",
+	"pclk_perihp_grf",
 };
 
 static const char *const rk3399_pmucru_critical_clocks[] __initconst = {
-- 
2.35.3

